### PR TITLE
DP-699 - Simplify DPF examples code based on API changes

### PR DESCRIPTION
**Pull request by gvjoshi25**

### CODE REVIEW

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java [hunk 1]
```diff
@@ -20,28 +21,32 @@
...
         DecisionTable table = new DecisionTable()
                 .addField(new CalculatedField("Variant Price", "toBigDecimal(${Variant Price})"))
                 
                 .addRule(new DecisionTableRule()
-                        .addCondition(new DecisionTableCondition("Variant Price", "? == null || ? < 20"))
-                        .addOutcome(new DecisionTableOutcome("Shipping", "0.00"))
-                        .addOutcome(new DecisionTableOutcome("Total", "${Variant Price} + Shipping"))
+                        .addCondition("Variant Price", "? == null || ? < 20")
+                        .addOutcome("Shipping", "0.00")
+                        .addOutcome(outcomes) // collection of outcomes and it can be reused if they are same for multiple rules.
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java]: @gvjoshi25 please drop this comment.

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java [hunk 2]
```diff
@@ -20,28 +21,32 @@
...
                         )
                 .addRule(new DecisionTableRule()
-                        .addCondition(new DecisionTableCondition("Variant Price", "? < 50"))
-                        .addOutcome(new DecisionTableOutcome("Shipping", "5.00"))
-                        .addOutcome(new DecisionTableOutcome("Total", "${Variant Price} + Shipping"))
+                        .addCondition("Variant Price", "? < 50")
+                        .addOutcome("Shipping", "5.00") // only provide variable & expression for outcome.
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java]: Please drop this comment.

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java (hunk 1 above)
Comment (gvjoshi25) [reply src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java]: removed.

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java (hunk 2 above)
Comment (gvjoshi25) [reply src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontable/AddADecisionTableToAPipeline.java]: removed.

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**